app/jobs/deny_restricted_verification_attempt_job.rb

```diff
@@ -0,0 +1,7 @@
+class DenyRestrictedVerificationAttemptJob < ApplicationJob
+  def perform(verification_attempt)
+    return unless verification_attempt.current_state == "restricted"
```

Comment:
By delaying this for a set period of time, we can make sure that restricted verification attempts automatically transition to denied (and they get the denial email) after a certain amount of time.

We could also leave the status in denied and delay only the sending of the denial _email_ if it feels better to have these stay in a different state forever, but I couldn't find a compelling reason to do that...